### PR TITLE
Update sample method docstring to describe shapes with more verbosity

### PR DESCRIPTION
**Pull request by phinate**
Adresses #60.

### CODE REVIEW

**Comment by imurray** [comment]
I assume the existing `...` parts in the comments are because we don't always assume the samples are necessarily a vector of length `features`, but could take other shapes. In which case replacing `...` with `features` would not be appropriate. @arturbekasov ?

File: nflows/distributions/base.py [hunk 1]
```diff
@@ -47,13 +47,16 @@ def sample(self, num_samples, context=None, batch_size=None):
 
         Args:
             num_samples: int, number of samples to generate.
-            context: Tensor or None, conditioning variables. If None, the context is ignored.
+            context: Tensor or None, conditioning variables. If None, the context is ignored. 
+                     Should have shape [context_size, context_features], which will generate
```
Comment (arturbekasov) [inline nflows/distributions/base.py:51]: Are we sure this isn't `[context_size, ...]`, i.e. context can't have an arbitrary shape?

**Comment by arturbekasov** [comment]
Hey @phinate, sorry for delay in response. Are you still interested in getting this merged in?

> Do you think it's enough to say something like "`[context_size, ...]`, where `...` represents an arbitrary shape feature vector"?

Yes, this sounds good.

**Comment by arturbekasov** [comment]
Thanks a lot, @phinate.

**Pull request merged**